nvme.c

```diff
@@ -564,6 +564,11 @@ static int get_smart_log(int argc, char **argv, struct command *cmd, struct plug
 	if (err)
 		return err;
 
+	if (is_blkdev(dev)) {
```

Comment:
This looks wrong. We should be able to get the smart log page data from a NVMe block device itself, right?

Reply:
As far I understand the specs says SMART is only for the controller not a namespace.